plugins/pluginTemplate/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "1",
+  "information": {
+    "name": "plugin_template",
```

Comment:
Change name to `plugin-template` and change plugin directory name to the same. I think it would be better looking if the directory name correponded with the plugin name, since it's the same way inside of UCRM.